src/workerd/api/streams/readable.h

```diff
@@ -296,6 +301,7 @@ class ReadableStream: public jsg::Object {
     JSG_METHOD(values);
 
     JSG_ASYNC_ITERABLE(values);
+    JSG_STATIC_METHOD(from);
```

Comment:
Shouldn't this be gated by the compat flag? I see you are throwing an exception when called, but it would be better to hide the method entirely, wouldn't it?

Reply:
Yeah that's probably better. The `JSG_TS_DEFINE` handling is a bit unfortunate in that case (as you'll see in the additional diff coming in a couple minutes)...

Reply:
Oof, I see.

Though I think, arguably, there's not a lot of value to writing TS overrides for this anyway. The official published workers-types bundle won't include them, since presumably we only publish the version built with no special flags set.

Meanwhile, I imagine the plan is that once the API is finalized, we will stop gating it behind a compat flag at all, since it's not a backwards-incompatible change. At that point the TS override could be updated to include this method, without duplicating them.

Reply:
(I am also fine with leaving it as you have it. Just proposing an alternative.)

Reply:
Let's leave it as is, but I do want to ping @mrbbot for awareness.

@mrbbot ... just noting here that the ts overrides get really messy when we have multiple compat flags in question. Not sure there's anything we can do about it reasonably but something to think about.